src/data-display/dynamic/dynamic-layout/templates/raw-table/index.vue

```diff
@@ -73,14 +72,9 @@ export default defineComponent<RawTableDynamicLayoutProps>({
         const state = reactive({
             fields: computed(() => {
                 if (state.rootData[0]) {
-                    const firstItem = state.rootData[0];
                     if (Array.isArray(props.options?.headers) && props.options?.headers?.length) {
-                        return props.options.headers.map((header) => ({
-                            key: header,
-                            name: header,
-                        }));
+                        return getSortingData(props.options.headers).map((value) => ({ key: value, name: value }));
                     }
-                    return sortBy(map(firstItem, (value, key) => ({ key, name: key })), (item) => item.key);
```

Comment:
issue: I think if this row is removed, it won't work correctly. 😮 Plz check the case that the headers are not given.